e2e_tests/test_workspace_services.py

```diff
@@ -20,7 +20,8 @@ async def test_create_guacamole_service_into_base_workspace(admin_token, verify)
             "description": "workspace for E2E",
             "address_space_size": "small",
             "client_id": f"{config.TEST_WORKSPACE_APP_ID}",
-            "client_secret": f"{config.TEST_WORKSPACE_APP_SECRET}"
+            "client_secret": f"{config.TEST_WORKSPACE_APP_SECRET}",
+            "app_service_plan_sku": f"{config.TEST_WORKSPACE_APP_PLAN}"
```

Comment:
I have something similar to this in my monitoring PR as I hit the same issue.

Rather than directly adding the `app_service_plan_sku`, I includeed this on the line after `payload` was initialised:

```python
    if config.APP_SERVICE_PLAN_SKU != "":
        payload["properties"]["app_service_plan_sku"] = config.APP_SERVICE_PLAN_SKU
 ```
 
My thinking was that this would skip it in the case where a value isn't provided by the env var and then the default from the porter config would be used?

Reply:
I've just pushed a change where I default the value in the config definition

Reply:
My thoughts are that we shouldn't really have that default in the porter.yaml - it should always be passed in